### PR TITLE
fixed edit page alignment on using input box

### LINKED ISSUE #103: Edit page alignment on using text box
On propose change pages,  text is not aligned properly. When we use text box. 

Screenshot_2021-04-10-15-42-57-51

Screenshot_2021-04-10-15-50-02-30

While this's ok for main page at least, but search bar isn't aligned as I addressed in other issue.

Screenshot_2021-04-10-15-47-32-70

Should any changes be made? For alignment.

Attributes name are also not visible. So this thing also,  should I address?

### PR DESCRIPTION
**Pull request by raashika03**
### Checklist

- [X] I indicated which issue (if any) is closed with this PR using [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [X] I only changed lines related to my PR (no bulk reformatting)
- [X] I indicated the source and check the license if I re-use code, or I did not re-use code
- [X] I made my best to solve only one issue in this PR, or explain why multi had to be solved at once.

### Issue

close #103

### Details
Can be seen live [here](https://raashika03.github.io/edam-browser/#data_0006)
Before:
IMG_20210504_042548

After:
IMG_20210504_042430

### CODE REVIEW

**Comment by raashika03** [comment]
IMG_20210504_044310
And if this kind of version is needed then desktop version is available on Android on which this problem generates.

File: edit.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
 <head>
     <title>EDAM ontology browser</title>
+    <meta name="viewport" content="width=device-width, user-scalable=no">
```
Comment (bryan-brancotte) [inline edit.html:5]: Well found :clap:. After reading some post on the matter, I found that scaling for people with "visual imparment" i.e have at least glasses. I propose that you use instead
```<meta name="viewport" content="width=device-width, initial-scale=1.0">```

From what I have tested it works fine, and keep the zoom ability.

**Review by bryan-brancotte — CHANGES_REQUESTED** [review]

File: edit.html (hunk 1 above)
Comment (raashika03) [reply edit.html]: Done @bryan-brancotte :) I hardly see any difference :sweat_smile:

Comment (raashika03) [reply edit.html]: Ok may be people with spects can spot any difference :sweat_smile:

**Comment by bryan-brancotte** [comment]
The support is not that good https://caniuse.com/mdn-http_headers_viewport-width but it is still a good option. I propose merge, and if one day someone find better we'll switch to it.

**Review by bryan-brancotte — APPROVED** [review]

**Comment by bryan-brancotte** [comment]
P.S: I also applied it to main page

**Comment by raashika03** [comment]
> P.S: I also applied it to main page

ok then I'll move that search icon to left side cause that's overlapping.

I tried solving #102 like this but I liked the desktop version more for mobile screen size for the main page, so was looking for some other approah if possible.

**Pull request merged**